schema.rst

```diff
@@ -43,6 +49,34 @@ Attribute names are case insensitive,
 although it is recommended that ``.cps`` files
 use the capitalization as shown.
 
+:attribute:`Action`
+-------------------
+
+:Type: |string| (restricted)
+:Applies To: |envvar|
+:Required: No
+
+This specifies how an environment variable's specified values
+are combined with any existing values of the environment variable.
+Note that "existing values" may also refer to values
+specified for the same environment variable by other components.
+
+- :string:`prepend`
+
+  The specified `Values`_ are inserted before
+  the environment variable's existing value(s), if any.
+  This is the default behavior.
+
+- :string:`append`
+
+  The specified `Values`_ are inserted after
+  the environment variable's existing value(s), if any.
+
+- :string:`replace`
+
+  The specified `Values`_ replace
+  the environment variable's existing value(s), if any.
+
```

Comment:
Looking at Meson's environment variable handling, this all looks pretty similar to what we do. I've been convinced that is a sane mechanism.

One thing that I'd like to see is the ability to completely unset an environment variable, since some tools treat an empty environment variable and an unset one differently. I see two ways that could be achieved, `replace` could be nullable, so that `replace: null` means "clear this from the environment", or there could be a `:bool:'clear'` attribute.